entities/schema/platformics.yaml

```diff
@@ -232,13 +237,20 @@ classes:
         range: FileUploadClient
       upload_error:
         range: string
+      created_at:
+        range: date
+        required: true
+        readonly: true
+      updated_at:
+        range: date
+        readonly: true
```

Comment:
This doesn't actually do anything since Files currently aren't codegenned, so changes in this file aren't reflected anywhere.

Robert already added the `created_at` and `updated_at` fields [to the model](https://github.com/chanzuckerberg/czid-platformics/blob/08191ccc0700665de2976a38f5054d24b861d761/entities/database/models/file.py#L62) a while back, I just updated it here for consistency.

Reply:
Also after discussing with Nina and Todd, we decided not to add a `deleted_at` column to Files.

Users can only access Files through an Entity, so there's not really a use case where we would want to mark a File for soft-deletion with the `deleted_at` column rather than just actually hard-deleting the File.